benchmark/data_frame_text_benchmark.py

```diff
@@ -166,6 +170,10 @@ def __init__(self, model: str):
             target_modules=target_modules,
         )
         self.model = get_peft_model(self.model, peft_config)
+        if model == 'roberta-large':
```

Comment:
Actually we don't use the pooling results not, instead we average over the last hidden states. Actually we can try to use the pooling layer results and also finetune the pooler.